hpx/runtime/serialization/serialize_sequence.hpp

```diff
@@ -1,173 +0,0 @@
-//  Copyright (c) 2007-2015 Hartmut Kaiser
```

Comment:
I'd vote for complete removal. We could ask Joel if he would like to accept a patch adding this directly to Boost, though.